sections/memory_layout.tex

```diff
@@ -69,15 +85,15 @@ \section{Block Packed Storage}\label{sec: memory layout}
 	\begin{algorithm}
 		\caption{Calculation of \(\lambda_n\) and Simulation of \(Z_{n+1}\)}	
 		\label{algo: calc lambda and simulate Z_{n+1}}
-		\(\xS \leftarrow \sigma_n\)\tcp*[f]{red cache}\;
+		\(\xS \leftarrow \sigma_n^2\)\tcp*[f]{red cache}\;
 		\(\xZ \leftarrow 0_{k\times 1}\)\tcp*[f]{blue cache}\;
 		\For{\(i = 0,\dots,n-1\)}{
-			\(\xl \leftarrow L_{\frac{n(n+1)}2+i}\)\tcp*[f]{teal cache}\;
-			\(\xS = \xS - \xl\xl^T\)\;
+			\(\xl \leftarrow L_{n,i}\)\tcp*[f]{teal cache}\;
+			\(\xS \leftarrow \xS - \xl\xl^T\)\;
 			\(\xY \leftarrow Y_i\)\tcp*[f]{violet cache}\;
-			\(\xZ = \xZ + \xl^T \xY\)\;
+			\(\xZ \leftarrow \xZ + \xl^T \xY\)\;
 		}
-		\(L_{\frac{n(n+1)}2 + n} \leftarrow \chol(\xS)\)\tcp*[f]{\(=L_{\frac{(n+2)(n+1)}2-1}\) i.e. on the diagonal}\;
+		\(L_{n,n} \leftarrow \chol(\xS)\)\tcp*[f]{lower triangular}\;
```

Comment:
Was hälst du von `L_{g(i)+k}` mit `g(n)=n(n+1)/2`. Dann sieht es fast so aus wie `L_{i,k}` man setzt das erste element in `g` ein und das zweite addiert man. Dann ist die blocknummerierung aber trotzdem noch das layout im speicher.